document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -29,18 +33,22 @@
     private static final Logger LOG = LoggerFactory.getLogger(MODULE_NAME_SPACE);
 
     @Override
-    public DocumentInfoResponse getDocumentInfo(DocumentInfoRequest documentInfoRequest) {
+    public DocumentInfoResponse getDocumentInfo(DocumentInfoRequest documentInfoRequest) throws DocumentGeneratorInterfaceException {
```

Comment:
The exception doesn't feel right. The interface itself isn't throwing this exception. Maybe DocumentInfoException would be more related to the issue which caused the exception?